restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnection.java

```diff
@@ -102,6 +102,14 @@
         DISCONNECTED,  /** Connection is in state disconnected */
     };
 
+    /**
+     * General status.
+     */
+    public enum Status {
```

Comment:
Don't worry bout that will fix it when I do the final merge. Please check my [last comment](https://github.com/RestComm/restcomm-android-sdk/pull/283#issuecomment-221534249) though